markdown/manual.md

```diff
@@ -1,220 +1,245 @@
 
-Any time that you won't be available during usual office hours (9 PM–5 PM)—for appointments, sick time, vacation, etc.—should be put on the calendar, and an email must be sent to the employee's supervisor.  For non-emergency medical absences, please notify your supervisor at least 48 hours in advance.
-
-We track time using Harvest. Hours worked must be recorded in Harvest by the end of each week (Sunday evening).
+Any time that you won't be available during the regular work day (9 PM to 5 PM locally) should be put on your calendar the organization's calendar.
 
 ### Vacation & Leave Policy
-We are closed on all Federal holidays and during the week between Christmas and the New Year.  We also have a few additional days off, please see [the attached holidays list](holidays.md).
+We are closed on all [Federal holidays](https://www.opm.gov/policy-data-oversight/pay-leave/pay-administration/fact-sheets/holidays-work-schedules-and-pay), the day after Thanksgiving, and during the week between Christmas and the New Year.
 
-We expect everyone to be able to get their work done in a timely fashion, and to not have to work on personal time.  Rested and refreshed people are better team members, and we strongly encourage enjoying life outside of the office.  Employees should take a *minimum* of two weeks of paid vacation per year, totaling 10 business days.
+We expect everyone to be able to get their work done in a timely fashion, and to not have to work on personal time.  Rested and refreshed people are better team members, and we strongly encourage enjoying life outside of the office.  
 
-To schedule time off, permission must be granted by your supervisor no less than 5 business days in advance, the dates and location of your travel must be put onto the team calendar, and the team should be notified via email.  To track time off, please track the vacation time in Harvest.
+*Employees should take a minimum of two weeks of paid vacation per year, totaling 10 business days.*
 
-For personal, medical, or family leave, each employee is encouraged to take the current Family and Medical Leave Act maximum.
+For personal, medical, or family leave, each employee is encouraged to take the current [Family and Medical Leave Act](https://www.dol.gov/general/topic/benefits-leave/fmla) maximum.
 
 For maternity and paternity leave, employees are encouraged to take up to 16 weeks of leave.
 
+#### Taking Leave
 Employees must inform your supervisor in writing as far in advance as is reasonable for staff and project planning purposes.  All extended periods of leave must be discussed with the Executive Director.
 
+To schedule time off, permission must be granted by your supervisor no less than 5 business days in advance, the dates your travel must be put onto the team calendar (just invite sayhello [at] opengovfoundation.org), and the team should be notified via email.
```

Comment:
@batemapf can we distinguish between "long vacation" (5 days in advance) versus "last minute opportunity to do something, or a family member/loved one is sick, or [insert life occurrence]" which is approval as soon as you can, as efficiently as you can (i.e. call the ED, etc)?